### ISSUE TITLE
Enable async backing for system parachains

### ISSUE THREAD

**Issue by s0me0ne-unkn0wn** [polkadot-fellows/runtimes#460]
# Overview

We had tried enabling 6-second block times for all the system parachains in #266, as it seemed to be easy, but it was discovered that this is blocked by paritytech/polkadot-sdk#3268. Since, considering the current system parachains transaction traffic, such an enablement doesn't provide any real benefit for the network, that work has been put on hold.

This issue serves as a tracking issue, should anyone want to perform a transition for a single system parachain or a group of them.

# Current status
## Kusama
- [ ] AssetHub
- [ ] BridgeHub
- [ ] Coretime
- [ ] Encointer
- [x] People

## Polkadot
- [ ] AssetHub
- [ ] BridgeHub
- [ ] Collectives
- [ ] Coretime
- [ ] People

**Comment by clangenb** [comment]
Encointer is also on 6s in v1.3.3, since https://github.com/polkadot-fellows/runtimes/pull/462

**Comment by eskimor** [comment]
With the new slot based collator - please keep slot times to 12s (single collator will produce two blocks in a row).

**Comment by clangenb** [comment]
Can you please elaborate that? Does this affect encointer that is already at 6 seconds too?

**Comment by eskimor** [comment]
There is just no need to. When writing above message I also thought that shorter slot times affects censorship resistance, but @bkchr rightly just pointed out that these points become moot with elastic scaling anyways. Nevertheless, I would recommend for chains to keep slot time at 12s, if there is no need to reduce it, as increasing is harder than decreasing. So if in doubt, better to leave it as is. (No harm done for chains which already changed it, 6s are fine too.)

**Comment by muharem** [comment]
I checked the pallets of Asset Hub, in 2503 sdk release, all AH pallets using block numbers, have `BlockNumberProvider` type within it's config. We can setup it to use Relay Chain blocks.

**Comment by muharem** [comment]
Async Backing for Kusama Asset Hub 
https://github.com/polkadot-fellows/runtimes/pull/659

**Comment by skunert** [comment]
Just an update, @lexnv will keep  track of bringing async-backing to polkadot-assethub.

**Comment by skunert** [comment]
@lexnv While the averag block time seems to be okay, I am seeing a ton of blocks produced by the same collator every 12s. This is a bit weird, in the current configuration the unincluded segment should get filled relatively fast and then each collator should just build one block every 6s. This should be looked into before we continue with enablement on other chains.
